### DISCUSSION TITLE
Setting system time/clock

### DISCUSSION THREAD

**Discussion by imgalli** [avajs/ava#3059] [Q&A] [answered]
Hi, how would people recommend that we achieve the same as the following with Jest? Would be grateful for any help, thanks!

```
jest.useFakeTimers().setSystemTime(Date.utc(2022, 0, 1, 0, 0, 0').getTime());
```

**Comment by sindresorhus** [comment, marked as answer]
https://sinonjs.org/releases/latest/fake-timers/
